### PR TITLE
Add Google Analytics tracking

### PR DESCRIPTION
**Pull request by edwardloveall**
This uses an environment variable `GOOGLE_ANALYTICS_ID` to set the tracking code for a Google Analytics we have set up. This will be present if two conditions are met:

* The GOOGLE_ANALYTICS_ID environment variable is set
* The rails environment is production

### CODE REVIEW

File: app/views/application/_javascript.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <%= yield :javascript %>
 
-<%= render "analytics" %>
+<%= render 'analytics' if Rails.env.production? %>
```
Comment (jneen) [inline app/views/application/_javascript.html.erb]: could we maybe check for `GOOGLE_ANALYTICS_ID` here instead?

Comment (jneen) [inline app/views/application/_javascript.html.erb]: (or just leave it out, since the whole partial is wrapped in `if`)

**Comment by jneen** [comment]
otherwise, :+1:

File: app/views/application/_javascript.html.erb (hunk 1 above)
Comment (edwardloveall) [inline app/views/application/_javascript.html.erb]: Yeah it's a bit verbose for sure. I wanted to make sure that A) we weren't sending analytics data in anything but production, and B) not adding dead javascript code if the `GOOGLE_ANALYTICS_ID` wasn't set.

**Pull request merged**